presto-hive/src/test/java/com/facebook/presto/hive/TestMergeJoinPlan.java

```diff
@@ -175,36 +187,87 @@ public void testMultipleJoinKeys()
 
         try {
             queryRunner.execute("CREATE TABLE test_join_customer5 WITH ( \n" +
-                    "  bucket_count = 4, bucketed_by = ARRAY['custkey', 'phone'], \n" +
-                    "  sorted_by = ARRAY['custkey', 'phone'], partitioned_by=array['ds'], \n" +
+                    "  bucket_count = 4, bucketed_by = ARRAY['custkey', 'nationkey'], \n" +
+                    "  sorted_by = ARRAY['custkey', 'nationkey'], partitioned_by=array['ds'], \n" +
                     "  format = 'DWRF' ) AS \n" +
                     "SELECT *, '2021-07-11' as ds FROM customer LIMIT 1000\n");
 
             queryRunner.execute("CREATE TABLE test_join_order5 WITH ( \n" +
-                    "  bucket_count = 4, bucketed_by = ARRAY['custkey', 'clerk'], \n" +
-                    "  sorted_by = ARRAY['custkey', 'clerk'], partitioned_by=array['ds']) AS \n" +
+                    "  bucket_count = 4, bucketed_by = ARRAY['custkey', 'orderkey'], \n" +
+                    "  sorted_by = ARRAY['custkey', 'orderkey'], partitioned_by=array['ds']) AS \n" +
                     "SELECT *, '2021-07-11' as ds FROM orders LIMIT 1000");
 
-            // merge join can't be enabled
+            // merge join can be enabled
             assertPlan(
                     mergeJoinEnabled(),
-                    "select * from test_join_customer5 join test_join_order5 on test_join_customer5.custkey = test_join_order5.custkey and test_join_customer5.phone = test_join_order5.clerk",
-                    joinPlan("test_join_customer5", "test_join_order5", ImmutableList.of("custkey", "phone"), ImmutableList.of("custkey", "clerk"), true));
+                    "select * from test_join_customer5 join test_join_order5 on test_join_customer5.custkey = test_join_order5.custkey and test_join_customer5.nationkey = test_join_order5.orderkey",
```

Comment:
joining nationkey and orderkey is confusing for people knowing the TPCH table schemas, because nation and order id are two completely different things.
There are two options to avoid the confusion.
1. Pick two TPCH tables that can be joined with multiple keys.
2. If you just want to use the TPCH tables as input and don't care about the meaning of the columns, rename the columns to something like key1, key2 in the created tables.